cmd/server/config/config.go

```diff
@@ -59,6 +63,11 @@ func (c *WhiteListConfig) Validate() error {
 			return errors.New("could not accept blank path as process white list")
 		}
 	}
+	for _, proxyPort := range c.ProxyPorts {
+		if proxyPort < 0 || proxyPort > 0xFFFF {
```

Comment:
What does this check ensure?

Reply:
a valid port is provided. Ports are limited to 16 bit integers, but the input takes in any size int

Reply:
Nice. Glad to see a thorough check lol